var/spack/repos/builtin/packages/zstd/package.py

```diff
@@ -33,34 +33,50 @@ class Zstd(MakefilePackage):
     version('1.1.2', sha256='980b8febb0118e22f6ed70d23b5b3e600995dbf7489c1f6d6122c1411cdda8d8')
 
     variant('programs', default=False, description='Build executables')
+    variant('libs', default='shared,static', values=('shared', 'static'),
+            multi=True, description='Build shared libs, static libs or both')
+    variant('compression', default='none', values=('none', 'zlib', 'lz4', 'lzma'),
```

Comment:
Should this be a multi-value variant?

Reply:
thanks for pointing that out

Reply:
Er also is it necessary to have "none" explicitly, or can the set of selected options be empty?

Reply:
If I understand the semantics correctly, I think we can use `any_combination_of('lz4', 'lzma', 'zlib')` here. Then `none` will be mutually exclusive with any combination of the other three values.

Reply:
Thanks, added that!